### PR TITLE
Long play mode and profile 5 RPUs generating

### PR DESCRIPTION
**Pull request by saindriches**

### CODE REVIEW

**Comment by quietvoid** [comment]
You need to run rustfmt for the CI.

**Comment by saindriches** [comment]
Done, now it passes checks.

File: dolby_vision/src/rpu/generate.rs [hunk 1]
```diff
@@ -26,11 +26,16 @@ pub struct GenerateConfig {
...
     /// Profile to generate
+    ///  - 5: IPT base layer with no reshaping
     ///  - 8.1: HDR10 base layer
     ///  - 8.4: HLG base layer with static reshaping (iPhone 13 MMR)
     #[cfg_attr(feature = "serde_feature", serde(default))]
     pub profile: GenerateProfile,
 
+    /// Set scene cut flag for every frame.
+    #[cfg_attr(feature = "serde_feature", serde(default))]
+    pub long_play_mode: bool,
```
Comment (quietvoid) [inline dolby_vision/src/rpu/generate.rs:37]: This would override the config since clap defaults flags to false.
It should be an Option like profile.

**Comment by saindriches** [comment]
Yes I didn't see a notification for that, will check it again later, thanks

File: src/commands/generate.rs [hunk 2]
```diff
@@ -65,4 +65,7 @@ pub struct GenerateArgs {
 
     #[clap(arg_enum, short = 'p', long, help = "Dolby Vision profile to generate")]
     pub profile: Option<GeneratorProfile>,
+
+    #[clap(long, help = "Set scene cut flag for every frame")]
+    pub long_play_mode: bool,
```
Comment (quietvoid) [inline src/commands/generate.rs:70]: This would override the config since clap defaults flags to false.
It should be an Option like profile.

**Pull request merged**